packages/xo-web/src/xo-app/user/index.js

```diff
@@ -172,6 +172,9 @@ class UserFilters extends Component {
                 if (!labelId) {
                   return
                 }
+                if (labelId === 'homeTypeVmTemplate') {
+                  type = 'VM-template'
+                }
```

Comment:
IMO it would make more sense for `home-filters` to use the correct type names. So it would export something like this:
```js
export default {
  VM,
  host,
  pool,
  'VM-template': vmTemplate,
  SR
}
```
and here, we'd simply import:
```js
import homeFilters from 'home-filters'
```
If you do that, don't forget to also change the import in `home/index.js`.